func_adl_xAOD/template/cms/r5/runner.sh

```diff
@@ -98,7 +98,7 @@ if [ $run = 1 ]; then
       fi
     fi
     export CMS_OUTPUT_FILE=ANALYSIS.root
-
+    
```

Comment:
This looks like an extra tab added here - this file shouldn't even be in this PR, I think.